OrbitGl/CaptureWindow.cpp

```diff
@@ -639,57 +618,50 @@ void CaptureWindow::RenderUI() {
   if (GetPickingMode() != PickingMode::kNone) {
     return;
   }
+}
 
-  ScopeImguiContext state(im_gui_context_);
-  Orbit_ImGui_NewFrame(this);
-
-#define VAR_TO_STR(var) VariableToString(#var, var)
-
-  if (draw_stats_) {
-    ImGui::ShowDemoWindow();
-    if (time_graph_.GetLayout().DrawProperties()) {
-      NeedsUpdate();
+void CaptureWindow::RenderImGui() {
+  if (ImGui::BeginTabBar("CaptureWindowTabBar", ImGuiTabBarFlags_None)) {
+    if (ImGui::BeginTabItem("Layout Properties")) {
+      if (time_graph_.GetLayout().DrawProperties()) {
+        NeedsUpdate();
+      }
+      ImGui::EndTabItem();
     }
 
-    m_StatsWindow.Clear();
-
-    m_StatsWindow.AddLine(VAR_TO_STR(width_));
-    m_StatsWindow.AddLine(VAR_TO_STR(height_));
-    m_StatsWindow.AddLine(VAR_TO_STR(world_height_));
-    m_StatsWindow.AddLine(VAR_TO_STR(world_width_));
-    m_StatsWindow.AddLine(VAR_TO_STR(world_top_left_x_));
-    m_StatsWindow.AddLine(VAR_TO_STR(world_top_left_y_));
-    m_StatsWindow.AddLine(VAR_TO_STR(world_min_width_));
-    m_StatsWindow.AddLine(VAR_TO_STR(mouse_x_));
-    m_StatsWindow.AddLine(VAR_TO_STR(mouse_y_));
-    m_StatsWindow.AddLine(VAR_TO_STR(time_graph_.GetNumDrawnTextBoxes()));
-    m_StatsWindow.AddLine(VAR_TO_STR(time_graph_.GetNumTimers()));
-    m_StatsWindow.AddLine(VAR_TO_STR(time_graph_.GetThreadTotalHeight()));
-
-    m_StatsWindow.AddLine(VAR_TO_STR(
-        GOrbitApp->GetCaptureData().GetCallstackData()->callstack_events_by_tid().size()));
-    m_StatsWindow.AddLine(
-        VAR_TO_STR(GOrbitApp->GetCaptureData().GetCallstackData()->GetCallstackEventsCount()));
-
-    m_StatsWindow.Draw("Capture Stats", &draw_stats_);
-  }
-
-#undef VAR_TO_STR
-
-  if (draw_help_) {
-    RenderHelpUi();
+    if (ImGui::BeginTabItem("Capture Info")) {
+      IMGUI_VAR_TO_TEXT(screen_width_);
+      IMGUI_VAR_TO_TEXT(screen_height_);
+      IMGUI_VAR_TO_TEXT(world_height_);
+      IMGUI_VAR_TO_TEXT(world_width_);
+      IMGUI_VAR_TO_TEXT(world_top_left_x_);
+      IMGUI_VAR_TO_TEXT(world_top_left_y_);
+      IMGUI_VAR_TO_TEXT(world_min_width_);
+      IMGUI_VAR_TO_TEXT(mouse_screen_x_);
+      IMGUI_VAR_TO_TEXT(mouse_screen_y_);
+      IMGUI_VAR_TO_TEXT(mouse_world_x_);
+      IMGUI_VAR_TO_TEXT(mouse_world_y_);
+      IMGUI_VAR_TO_TEXT(time_graph_.GetNumDrawnTextBoxes());
+      IMGUI_VAR_TO_TEXT(time_graph_.GetNumTimers());
+      IMGUI_VAR_TO_TEXT(time_graph_.GetThreadTotalHeight());
+
+      IMGUI_VAR_TO_TEXT(
+          GOrbitApp->GetCaptureData().GetCallstackData()->callstack_events_by_tid().size());
+      IMGUI_VAR_TO_TEXT(GOrbitApp->GetCaptureData().GetCallstackData()->GetCallstackEventsCount());
+
+      ImGui::EndTabItem();
+    }
 
-    if (first_help_draw_) {
-      // Redraw so that Imgui resizes the
-      // window properly on first draw
-      NeedsRedraw();
-      first_help_draw_ = false;
+    if (ImGui::BeginTabItem("Misc")) {
```

Comment:
About this "Misc" tab:

- Why we have the button and present the view as a separate window?
- At least in my case, the pop up window starts with a bigger size than it has to be (a lot of empty space), do you know why could it be?